docs/reference/setup/jvm-options.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[jvm-options]]
+=== Setting JVM options
+
+The preferred method of setting Java Virtual Machine options (including
+system properties and JVM flags) is via the `jvm.options` configuration
+file. The default location of this file is `config/jvm.options` (when
+installing from the tar or zip distributions) and
+`/etc/elasticsearch/jvm.options` (when installing from the Debian or RPM
+packages).
+
```

Comment:
This syntax is pretty funky as the first thing that users need to read about configuring ES.  I'd add a section at the top of this page saying "You probably only need to set the heap size [link] in this file, but in case you need to configure other JVM options..."

Reply:
I pushed 73fc33030f20ea8cef43ac4ef81a01eab96ab768.